src/libratbag.c

```diff
@@ -345,6 +345,9 @@ ratbag_driver_fallback_logitech(struct ratbag_device *device,
 	if (!rc)
 		rc = ratbag_test_driver(device, dev_id, "hidpp10", NULL);
 
+	if (!rc)
+		rc = ratbag_test_driver(device, dev_id, "logitech_g300", NULL);
+
```

Comment:
Not so sure we need to have autodetection if we need to put some profile information in the driver...